index.html

```diff
@@ -9215,6 +9262,47 @@ <h3><dfn>Take Element Screenshot</dfn></h3>
  <li><p>Return <a>success</a> with data <var>encoded string</var>.
 </ol>
 </section> <!-- /Take Element Screenshot -->
+
+<section>
+<h3><dfn>Take Full Page Screenshot</dfn></h3>
```

Comment:
Please move before `Take Element Screenshot` because it's closer related to `Take Screenshot`.

Reply:
Done